python/src/MetaModelValidation_doc.i.in

```diff
@@ -1,7 +1,9 @@
 %feature("docstring") OT::MetaModelValidation
-"Base class to score a metamodel and perform validations.
+"Score a metamodel and perform validation.
+
+Available constructors:
+    MetaModelValidation(*outputValidationSample, outputPredictionSample*)
```

Comment:
you can drop this, and retain only the new arguments in the Parameters section

Reply:
We have to keep this ctor, because the other one is deprecated after this PR.

Reply:
I meant that you could remove the "Available constructors:"  section, and keep only the arguments of the new ctor in the Parameters section because the old one is deprecated

Reply:
Ok. Fixed at 0d8bfddb6.